webapi/src/main/twirl/queries/sparql/v1/getResourceInfo.scala.txt

```diff
@@ -50,7 +50,7 @@ WHERE {
     {
         @* Query statements about the resource's file values. *@
 
-        ?prop rdfs:subPropertyOf* knora-base:hasFileValue .
+        ?prop rdfs:subPropertyOf+ knora-base:hasFileValue .
```

Comment:
This implies that `knora-base:hasFileValue` is never used directly, but only by means of subproperties. Correct?